src/main/scala/cognite/spark/AssetsRelation.scala

```diff
@@ -113,5 +113,5 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
 
   override def toRow(a: Asset): Row = asRow(a)
 
-  override def uniqueId(a: Asset): Long = a.id
+  override def uniqueId(a: Asset): String = a.id.toString
```

Comment:
Feels like we shouldn't need this if we just generalize over the return value type, somewhere?

Reply:
We could add it as a generic type to `SdkV1Relation` and `SdkV1Rdd`, but wanted to avoid another generic there for something so small. Long can always be cast to String, which would you prefer?